src/org/labkey/test/components/react/DropdownButtonGroup.java

```diff
@@ -281,6 +286,12 @@ public DropdownButtonGroupFinder withButtonId(String Id)
             return this;
         }
 
+        public DropdownButtonGroupFinder withButtonClass(String Id)
+        {
+            _baseLocator = Locators.buttonGroupWithClass(Id);
```

Comment:
seems a little confusing to pass an Id to a method that advertises taking a class

Reply:
I'll update the be variable to be cls